src/components/Pricing/constants.tsx

```diff
@@ -21,6 +21,7 @@ export const features = {
     ],
     'Advanced features': [
         { title: 'Correlation analysis' },
+        { title: 'Group Analytics' },
         { title: 'Priority Session Recording' },
         { title: 'Multivariate testing' },
         { title: 'Deep Dive Dashboards' },
```

Comment:
```suggestion
```

Reply:
Happy to do this, but have 0 context? If you're confident, just commit this suggestion.

Reply:
We don't actually have this feature so was just a tiny bit of housekeeping